pages/signup.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div>signup</div>
```

Comment:
* 머지 하시고 기능 추가 부탁드립니다
```suggestion
<div>signup - 추가</div>
```